recipes/vscode-python/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "vscode-python" %}
+{% set version = "2021.2.582707922" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 517360d2af77dc7376d02cdfdf9348142d1d2c7b792e5ff6c548871b2aa4e991
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - nodejs 12.14.1
+    - code-server >=3.9
+  host:
+    - nodejs 12.14.1
+    - code-server >=3.9
+  run:
+    - code-server >=3.9
+    - python >=2.7
+    - jedi <0.18
+    - pygls  # [py>=36]
+    - jedi-language-server <0.22  # [py>=36]
+    - isort 5.7.0  # [py>=36]
+
+
+
+test:
+  commands:
+    - code-server --list-extensions --show-versions | grep ms-python.python@2021.2.1
```

Comment:
unfortunately the release version and the version of the installed version as not equal and I believe it's not possible to infer one from the other.